pages/Admins/index.tsx

```diff
@@ -1,8 +1,28 @@
+import * as React from 'react';
 import Layout from "../../components/Layout/Layout";
+import styles from './Admins.module.css';
+import { Grid, Button, Box } from "@mui/material";
 
 const AdminPage: React.FunctionComponent = () => {
+
     return(
-        <Layout title='Admins'/>
+      <main>
```

Comment:
Did you mean to enclose everything in this component within a `main` tag? Conventionally, we should just use one main tag per page if we do use them. (I also pulled this branch and saw that there wasn't really a difference if outermost main tag was removed)